hack/build-wheels.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+ROOT_FOLDER="$(dirname "${0}")/.."
```

Comment:
:thought_balloon: I know this form is used elsewhere, but a nice, reliable alternative for finding the root directory of a git repo is `git rev-parse --show-toplevel`